docs/user/rollout-policies.md

```diff
@@ -0,0 +1,107 @@
+
+- **Labels**: Devices with specific metadata labels can be targeted for rollouts.
+- **Fleet Membership**: Rollouts apply only to devices within the specified fleet.
+
+### **Device Selection Strategy**
+
+Currently, Flightctl supports only the **BatchSequence** strategy for device selection. This strategy defines a stepwise rollout process where devices are added in batches based on specific criteria.
+
+Batches are executed sequentially. After each batch completes, execution proceeds to the next batch only if the success rate of the previous batch meets or exceeds the configured **success threshold**. The success rate is determined as:
+
+```text
+# of successful rollouts in the batch / # of devices in the batch >= success threshold
+```
+
+A **successful rollout** is when a device completes the rollout process successfully.
+
+### **Limit in Device Selection**
+
+Each batch in the **BatchSequence** strategy uses a `limit` parameter to define how many devices should be included in the batch. The limit can be specified in two ways:
+
+- **Absolute number**: A fixed number of devices to be selected.
+- **Percentage**: A percentage of the total device population that matches the label criteria.
+  - If a `selector` with labels is provided, the percentage is calculated based on the number of devices that match the label criteria within the fleet.
+  - If no `selector` is provided, the percentage is applied to all devices in the fleet.
+
+### **Success Threshold**
+
+The **success threshold** defines the percentage of successfully updated devices required to continue the rollout. If the success rate falls below this threshold, the rollout may be paused or rolled back to prevent further failures.
+
+### **Example YAML Configuration (Fleet Spec)**
+
+```yaml
+apiVersion: v1alpha1
+kind: Fleet
+metadata:
+  name: default
+spec:
+  selector:
+    matchLabels:
+      fleet: default
+  rolloutPolicy:
+    deviceSelection:
+      strategy: 'BatchSequence'
+      sequence:
+        - selector:
+            matchLabels:
+              site: madrid
+          limit: 1  # Absolute number
+        - selector:
+            matchLabels:
+              site: madrid
+          limit: 80%  # Percentage of devices matching the label criteria within the fleet
+        - limit: 50%  # Percentage of all devices in the fleet
+        - selector:
+            matchLabels:
+              site: paris
+        - limit: 80%
+        - limit: 100%
+    successThreshold: 95%
+```
+
+---
+
+## Rollout Disruption Budget
+
+### **Definition**
+
+A rollout disruption budget defines the acceptable level of service impact during a rollout. This ensures that a deployment does not take down too many devices at once, maintaining overall system stability.
+
+### **Disruption Budget Parameters**
+
+1. **Group By**: Defines how devices are grouped when applying the disruption budget.
```

Comment:
We should indicate that the "groupBy" entries should match label keys within the fleet devices.

An maybe provide a detailed example out of the YAML definition below (eg. define what groups exist, and that the budget refers to each of them independently).

Reply:
OK